src/main/java/no/sikt/nva/BrageProcessor.java

```diff
@@ -106,4 +112,19 @@ private Optional<Record> processBundle(LicenseScraper licenseScraper,
             return Optional.empty();
         }
     }
+
+    private URI getHandle(File entryDirectory, DublinCore dublinCore) throws HandleException {
+        return noHandleCheck ?
+                   getHandleAndIgnoreErrors(entryDirectory, dublinCore)
+                   : handleScraper.scrapeHandle(getHandlePath(entryDirectory), dublinCore);
+    }
+
+    private URI getHandleAndIgnoreErrors(File entryDirectory, DublinCore dublinCore) {
+        try {
+            return handleScraper.scrapeHandle(getHandlePath(entryDirectory), dublinCore);
+        } catch (HandleException e) {
+            //ignored
+            return null;
+        }
+    }
```

Comment:
Er ikke det bedre å catche exception her når vi ikke ønsker å ha handlecheck? For da slipper vi å opprette en record med "null" handle.

   public URI scrapeHandle(final Path handlefile, final DublinCore dublinCore) throws HandleException {
        try {
            var optionalHandle = getHandleFromInTitlesAndHandlesMap(dublinCore);
            if (optionalHandle.isPresent()) {
                return optionalHandle.get();
            } else {
                return extractHandleFromBundle(handlefile, dublinCore);
            }
        } catch (Exception e) {
            throw new HandleException(COULD_NOT_FIND_HANDLE_IN_HANDLE_FILE_NOR_DUBLIN_CORE_OR_IN_SUPPLIED_CSV, e);
        }
    }

Reply:
Men det har ikke så mye å si for min del. Eller er det for "testing purpose"?

Reply:
Slik jeg forstod det var det for å få lagd record selv om handle manglet slik at resten av valideringa skjedde. Dette er kun for test kjøring for å detektere flest mulig feil. Usikker på hvor viktig denne funksjonaliteten er nå som vi har fått alle NVE postene med handles.

Reply:
Skjønner, bare å merge :)